tsconfig.json

```diff
@@ -2,10 +2,17 @@
   "extends": "./node_modules/gts/tsconfig-google.json",
   "compilerOptions": {
     "rootDir": ".",
-    "outDir": "build"
+    "outDir": "build",
+    "resolveJsonModule": true,
+    "lib": [
+      "es2016",
```

Comment:
es2018!

Reply:
Isn't this something synthtool would generate, though?

Reply:
generator will generate `es2018` here, but I think we are excluding this file because of the manual layer. Just need to fix it here manually.

Reply:
I will update to 'es2018'

Reply:
Still says 2016 :)

Reply:
ping: `es2016` still stays :)